test/server.cpp

```diff
@@ -268,18 +251,12 @@ TEST_F(ServerTest, 400)
 const char* urls404[] = {
   "/",
   "/zimfile",
-  "/ROOT/non-existent-item",
```

Comment:
Shouldn't we check that all those removed urls404 are now redirections ?
(Maybe not all as they may now fall in a common scheme, but at least few of them)

Reply:
Good suggestion. I took advantage of it to improve the `ServerTest.NonEndpointUrlsAreRedirectedToContentUrls` unit-test even more.